package.json

```diff
@@ -45,9 +45,9 @@
     "prop-types": "^15.6.0",
     "query-string": "^6.3.0",
     "rc-slider": "^8.6.1",
-    "react": "^16.3.2",
+    "react": "16.8.6",
```

Comment:
pinning `react` `react-dom` and `react-test-renderer` to 16.8.6 since from 16.9.0 there will be warnings for legacy component lifecycles and that will cause tests to fail. See https://github.com/facebook/react/releases/tag/v16.9.0 and https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html

Pinning react is just a temp solution. We should spend time to work on deprecating those unsafe component lifecycles